apps/penxle.com/src/lib/tiptap/node-views/access-barrier/Component.svelte

```diff
@@ -34,6 +36,11 @@
       }
     }
   `);
+
+  const handlePriceChange = (e: Event) => {
+    const price = Number((e.currentTarget as HTMLInputElement).value);
```

Comment:
<img width="631" alt="스크린샷 2023-11-20 오후 5 56 05" src="https://github.com/penxle/penxle/assets/5278201/e654ec44-ce5e-4a4d-aa64-6e5535eeaf96">

P1: 직접 입력해보니 NaN 입력이 되어 해당 경우 입력 차단이 필요해보입니다.

```suggestion
    const price = e.currentTarget.valueAsNumber
		if (Number.isNaN(price)) return
```